etc/profile-m-z/nodejs-common.profile

```diff
@@ -7,9 +7,26 @@ include nodejs-common.local
 # added by caller profile
 #include globals.local
 
+# Add the next line to your nodejs-common.local if you prefer to disable gatsby telemetry.
+#env GATSBY_TELEMETRY_DISABLED=1
+
 blacklist /tmp/.X11-unix
 blacklist ${RUNUSER}
 
+ignore read-only ${HOME}/.npm-packages
+ignore read-only ${HOME}/.npmrc
+ignore read-only ${HOME}/.nvm
+ignore read-only ${HOME}/.yarnrc
```

Comment:
Maybe prefer `read-write`? IDK

Reply:
> Maybe prefer `read-write`? IDK

I agree that `read-write` sounds a lot more logical than the `ignore blah` construct. Just checked and I used it in git-cola, gnome-{logs,system-log} and yelp too apparently. Can't remember why OTOH but I'll change it after testing it gives the same result. That'll be over the weekend though, I'm clocking off for today. Thanks for the suggestions, much appreciated. And enjoy the weekend!

Reply:
FWIW: The only case where `ignore read-only` should be used instead of `read-write` are redirect profiles IMHO.